### PR TITLE
[NOJIRA] Updated local deployment documentation for Cloud LRS

### PR DESCRIPTION
**Pull request by sandeep-jay**
Updated local deployment documentation for Cloud LRS. Updated the deploy/start.sh script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Cloud-LRS
 
-Cloud-LRS is a Learning Record Store, a central place for collecting and storing interoperable Learning Activities from multiple campus systems.
-It can store statements in compliance with xAPI and IMS Caliper formats.
+Cloud-LRS is a Learning Record Store, a central place for collecting and storing interoperable Learning Activities from multiple campus systems. It can store statements in compliance with xAPI(v1.0) and IMS Caliper(v1.1) formats.
```
Comment (johncrossman) [inline README.md:4]: @sandeepmjay Nit-picky concern: Does "Cloud-LRS" need a dash in this context?

Comment (sandeep-jay) [reply README.md:4]: Not nit picky at all. Pushing amends on this :)

Comment (raydavis) [reply README.md:4]: @johncrossman @sandeepmjay Since I've merged & I'm about to submit a PR with additional changes, how about I remove the dash in mine?

Comment (johncrossman) [reply README.md:4]: Sounds good, @raydavis

**Pull request merged**